TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleMain.java

```diff
@@ -47,16 +71,40 @@ public void initialize() {
         m_driverOp = new GamepadEx(gamepad1);
         dashboard = FtcDashboard.getInstance();
 
+        //FlickerAction
+        flickerAction = new TimedAction(
+                ()-> flicker.setPosition(0.27),
+                ()-> flicker.setPosition(0.5),
+                500,
+                true
+        );
+
         //Subsystems and Commands
         driveSystem = new DriveSystem(fL, fR, bL, bR);
         driveCommand = new Com_Drive(driveSystem, m_driverOp::getLeftX, m_driverOp::getLeftY, m_driverOp::getRightX, ()->mult);
+
+        shooterSystem = new ShooterSubsystem(flyWheel, flicker, flickerAction, telemetry);
+        shooterCommand = new Com_Shooter(shooterSystem);
+        runFlyWheelCommand = new InstantCommand(()->flyWheel.set(1.0), shooterSystem);
+        shootCommandGroup = new SequentialCommandGroup(runFlyWheelCommand,
+                new WaitCommand(1000),shooterCommand);
+
+        intakeSystem = new IntakeSubsystem(intakeA);
+        intakeCommand = new Com_Intake(intakeSystem);
+        outtakeCommand = new Com_Outtake(intakeSystem);
+
 //       Old Method no longer necessary:
 //        slowDrive = new GamepadButton(m_driverOp, GamepadKeys.Button.Y)
 //                .toggleWhenPressed(()->mult = 0.5, ()->mult = 1.0);
 
         m_driverOp.getGamepadButton(GamepadKeys.Button.Y)
                 .toggleWhenPressed(()->mult = 0.5, ()->mult = 1.0);
 
+        m_driverOp.getGamepadButton(GamepadKeys.Button.A).toggleWhenActive(shootCommandGroup);
```

Comment:
Why is this toggle when active? It should be a `whileHeld`.

Reply:
The shooter command doesn't end because it doesn't have an `isFinished()`, so this sequential command group will never end.

Reply:
You need to make your own SequentialCommandGroup subclass that overrides `end(interrupted)` and stops the shooter when it gets let go. And then make the binding `whenHeld`.